### PR TITLE
Thellimist/pi 845

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

**Comment by thellimist** [comment]
This check is added to `HomePage`.  

I had two options 
1) add to `asyncZoneFetchSettings` and `asyncZoneUpdateSetting` or
2) add to a renderable page (choose homepage).

The drawback of this approach is if we move `DevelopmentModeCard` to another page except for HomePage the notification will not be shown instantly. It'll only be shown when `HomePage` is rendered. If we always keep `DevelopmentModeCard` in `HomePage` we will get instant notifications.

File: lang/en.js [hunk 1]
```diff
@@ -221,5 +221,6 @@
     "constants.plans.ent": "Enterprise plan",
     "errors.noActiveZoneSelected": "Please select a domain that is provisioned with CloudFlare.",
     "warning.usingSubdomain": "You are using a subdomain for your site, but any CloudFlare settings applied via this plugin will be applied to your original domain as well",
+    "warning.developmentmode": "Development mode enabled, all traffic will bypass the Cloudflare cache",
```
Comment (jwineman) [inline lang/en.js]: Need a period on the end of this.

**Comment by jwineman** [comment]
This won't fire if they don't load the homepage.  Instead I would check the status of development mode like this:
src/js/actions/zoneSettings.js
```
export function asyncZoneFetchSettings(zoneId) {
    return dispatch => {
        dispatch(zoneFetchSettings());
        zoneGetSettings(zoneId, function(error, response){
            if(v4ResponseOk(response)) {
                //new
                if([DEVELOPMENT MODE IS ON]) {
                  dispatch(notificationAdd("warning.developmentmode",true,true));
                 }
                dispatch(zoneFetchSettingsSuccess(zoneId, response.body.result));
            } else {
                dispatch(notificationAddClientAPIError(zoneFetchSettingsError(), response));
            }
        });
    };
}
```
This functionality is app specific, not home page specific so we don't want the logic in the home page.

**Comment by thellimist** [comment]
If I put it in `asyncZoneFetchSettings` then when the user enables `Development Mode` "on" the notification will not be shown. That's why we need to edit two places `asyncZoneFetchSettings` and `asyncZoneUpdateSetting`

**Comment by jwineman** [comment]
Good catch.  Yeah do that.

**Pull request merged**